packages/react-native-executorch/common/runner/runner.cpp

```diff
@@ -83,7 +83,7 @@ Error Runner::load() {
   ET_CHECK_OK_OR_RETURN_ERROR(module_->load_method("forward"));
   // load tokenizer.
 
-  auto blob = loadBytesFromFile(tokenizer_path_);
```

Comment:
TODO: Remove definition of `loadBytesFromFile` from this file and add import of `rnexecutorch::file_utils::loadBytesFromFile` from `FileUtils.cpp`